selfdrive/controls/lib/drive_helpers.py

```diff
@@ -95,8 +95,8 @@ def update_v_cruise(v_cruise_kph, v_ego, gas_pressed, buttonEvents, button_timer
 
 def initialize_v_cruise(v_ego, buttonEvents, v_cruise_last):
   for b in buttonEvents:
-    # 250kph or above probably means we never had a set speed
-    if b.type in (ButtonType.accelCruise, ButtonType.resumeCruise) and v_cruise_last < 250:
+    # 0kph means we never had a set speed
+    if b.type in (ButtonType.accelCruise, ButtonType.resumeCruise) and v_cruise_last != 0:
```

Comment:
I don't think 0 is a much better magic value

Reply:
Well I liked 0 because we don't have to "estimate" what 255 converted to m/s would be in the car controllers (> 70?) as we only have access to the converted value.